Sources/PrivoSDK/PrivoAuth.swift

```diff
@@ -96,14 +96,20 @@ public class PrivoAuth {
     
     private let app: UIApplication
     private let userDefaults: UserDefaults
-    private let api: Rest
+    private let api: IRest
```

Comment:
we do not use pattern with I prefixes in protocols, etc

Reply:
renamed